bindings/wallet-core/src/vote.rs

```diff
@@ -0,0 +1,42 @@
+use chain_impl_mockchain::{
+    certificate::{VoteCast, VotePlanId},
+    vote::{Choice, Options, Payload, PayloadType},
+};
+
+pub const VOTE_PLAN_ID_LENGTH: usize = 32;
+
+pub struct VotePlan {
+    id: VotePlanId,
+    payload_type: PayloadType,
+}
+
+pub struct Proposal {
+    index: u8,
+    options: Options,
+}
```

Comment:
I don't see the value to split them both in the API. Have them merged. Otherwise use the structure provided by `chain-libs`

Reply:
Okay